lib/vsc/utils/fancylogger.py

```diff
@@ -562,6 +586,28 @@ def _logToSomething(handlerclass, handleropts, loggeroption, enable=True, name=N
     return handler
 
 
+def _getScreenLogFormatter(stream, colorize='never'):
+    """
+    Return a log formatter, with optional colorization features.
+
+    Second argument `colorize` controls whether the formatter
+    can use ANSI terminal escape sequences:
+
+    * ``'never'`` (default) forces use of the plain `logging.Formatter` class;
```

Comment:
> > +    Second argument `colorize` controls whether the formatter
> > +    can use ANSI terminal escape sequences:
> > +
> > +    * ``'never'`` (default) forces use of the plain `logging.Formatter` class;
>
> `forces` -> `returns`

Hmm... I kind of disagree: both are of course correct, but "force" is
there to stress that no check is done whether outputting ANSI control
sequences to `stream` is sensible or not.